styles.css

```diff
@@ -0,0 +1,82 @@
+@import url('https://fonts.googleapis.com/css2?family=Alumni+Sans+Pinstripe&family=Poppins&display=swap');
+
+#toolbar {
+  display: flex;
+  position: fixed;
+  height: 35px;
+  width: 920px;
+  padding: 10px;
+}
+
+#logo {
+  color: #6070ff;
+  margin-right: 125px;
+  font-size: large;
+  font-weight: bold;
+}
+
+.nav-item {
+  display: flex;
+  margin-left: 10px;
+  font-family: 'Poppins', sans-serif;
+}
+
+#headline {
+  display: flex;
+  flex-direction: column;
+  height: 740px;
+  width: 360px;
+  background-image: url("assets/images/mobile.png");
+  padding: 10px;
+  margin-top: 114px;
+}
```

Comment:
- [ ] Good job using Flexbox to layout this section. Please you should stick to the design as much as possible as it is expected that your work looks exactly like the design. Kindly adjust the background image to match the design template.
- [ ] Kindly make sure that the spacing between elements matches the design, please.
*Design template*
![aaa](https://user-images.githubusercontent.com/94551484/185696437-f73549f8-3410-4c47-b422-66e5f42d13ff.png)
*Your design*
![Screenshot 2022-08-19 204640](https://user-images.githubusercontent.com/94551484/185696442-f789cd43-7c6d-4d34-93f5-07587c6ae6c1.png)
You can try removing the `margin-top` property and use this properties instead. `padding-top: 114px; background-repeat: no-repeat; background-position-x: center;`

Reply:
I have fixed spaces between items